Python/KernelDetector/EncodeDetect_new.py

```diff
@@ -59,9 +59,11 @@ def DetectKernels(sourceFile, thresh = 0.95, hotThresh = 512, newLine = False):
         for sub in blockMap[key]:
```

Comment:
This entire project should be deleted now

Reply:
Done